lib/ruby-next/commands/nextify.rb

```diff
@@ -108,6 +113,11 @@ def parse!(args)
           exit 2
         end
 
+        if @overwrite_original_file && !@single_version
```

Comment:
I think, we should use `single_version?` here and also mention `--rewriters` (as an alternative to `--single-version`) in the message:

```suggestion
        if overwrite_original_file? && !single_version?
```

Reply:
thanks!

done
Also added spec test for --overwrite and --rewrite args